Comments on manucript

TCLamnidis opened the pull request.

TCLamnidis committed:
Update 03.results-and-discussion.md

This punctuation reads more natural to me. Hopefully it doesn't alter the meaning that was meant?

TCLamnidis committed:
Update 03.results-and-discussion.md

Seems odd to have sub-points that follow the same naming as the super-points. Changed to "a, b, c, d".

TCLamnidis committed:
small typo

TCLamnidis committed:
Update 03.results-and-discussion.md

TCLamnidis committed:
Update 03.results-and-discussion.md

No other tool has a specific citing of coder, and it is a inconsistent to have one here.

TCLamnidis committed:
Update 03.results-and-discussion.md

Just a suggestion.makes it clearer what you mean, but perhaps it is too specific. never used this so nont sure if you would ever remove non-human data too.

TCLamnidis committed:
Deleted other references to people.

What every person did should be in an 'Author contributions' section at the end, not in main text. That is if you think having a specific contribution list stating who made what script is even necessary. 
Also, whether JAFY had computational training isn't really relevant to a reader. Makes JAFY more sound relatable, sure, but it is not of significance for the pipeline.

TCLamnidis committed:
Update 03.results-and-discussion.md

oxford comma. Pretty sure I saw another earlier on. best be consistent.

TCLamnidis committed:
Update 03.results-and-discussion.md

TCLamnidis committed:
Update 03.results-and-discussion.md

No need to name-call SoSci.

TCLamnidis committed:
Update 04-conclusion.md

TCLamnidis committed:
Update 05.methods.md

TCLamnidis committed:
Update 80-acknowledgements.md

TCLamnidis committed:
Update 80-acknowledgements.md

Added Stephan since he funded my attendance to the Nextflow Workshop in Tubingen. Not sure if this is what you had in mind. If you meant more hackathons, then feel free to delete this.

jfy133 commented on content/03.results-and-discussion.md:
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
-   2. Mapping quality filtering: Samtools (@doi:10.1093/bioinformatics/btp352)
-   3. PCR duplicate removal: DeDup (@doi:10.1186/s13059-016-0918-z), Picard MarkDuplicates(@doi:10.1101/gr.107524.110)
-   4. Mapping statistics generation: PreSeq (@doi:10.1038/nmeth.2375), Qualimap2(@doi:10.1093/bioinformatics/btv566)
+   a. Alignment against reference genome: BWA (@doi:10.1093/bioinformatics/btp324, @doi:10.1093/bioinformatics/btp698, @arXiv:1303.3997), CircularMapper (@doi:10.1186/s13059-016-0918-z)
```
Nope, this breaks markdown, please revert.

TCLamnidis replied:
OOF, my bad! Converted to bullet points so there aren't nested numbers.

jfy133 commented on content/03.results-and-discussion.md:
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
-In nf-core/eager, all tools also originally used in EAGER have been updated to latest versions, as available on Bioconda (@doi:10.1038/s41592-018-0046-7) and conda-forge (@url:https://conda-forge.github.io) to ensure widespread accessibility and stability of utilized tools. The MapDamage2 (@doi:10.1093/bioinformatics/btt193) and Schmutzi (@doi:10.1186/s13059-015-0776-0) methods have not been carr ...
+In nf-core/eager, all tools also originally used in EAGER have been updated to latest versions, as available on Bioconda (@doi:10.1038/s41592-018-0046-7) and conda-forge (@url:https://conda-forge.github.io) to ensure widespread accessibility and stability of utilized tools. The MapDamage2 (@doi:10.1093/bioinformatics/btt193) and Schmutzi (@doi:10.1186/s13059-015-0776-0) methods have not been carr ...
 
-We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2  ...
```
Why do you want to remove the initials?

TCLamnidis replied:
Is that the normal way to do this in bioinfo papers? seems strange to me that the specific scripts are attributed to collaborators in the main text. That is what the "Author contributions" section is for in papers.

apeltzer replied:
Yeah normally thats not used - either make people co-author (which all of them are) or have them for very minor stuff in acknowledgments ;-)

jfy133 replied:
Ok fair enough. Was trying to give more direct attribution to 'unpublished' tools, but I guess not necessary. Author contributions don't normally go into that much detail.

jfy133 commented on content/03.results-and-discussion.md:
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
+In nf-core/eager, all tools also originally used in EAGER have been updated to latest versions, as available on Bioconda (@doi:10.1038/s41592-018-0046-7) and conda-forge (@url:https://conda-forge.github.io) to ensure widespread accessibility and stability of utilized tools. The MapDamage2 (@doi:10.1093/bioinformatics/btt193) and Schmutzi (@doi:10.1186/s13059-015-0776-0) methods have not been carr ...
 
-We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2  ...
+We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2  ...
```
```suggestion
We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2 (@doi:10.1186/s13059-019-1891-0). Characterisation of properties of authentic ancient DNA (@doi:10.1073/pnas.0704665104) from MALT alignments is carried out with the HOPS pipeline (@doi:10.1186/s13059-019-1903-0). Ancient metagenomic studies sometimes may include comparative samples from living individuals (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data privacy, nf-core/eager includes a 'strip_fastq' script which creates raw FASTQ files but with reference-genome mapped reads removed. This then allows safe upload of sequencing data to public repositories with identifiable human data removed.
```

jfy133 commented on content/03.results-and-discussion.md:
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
 ## Accessibility
 
-Alongside the portable new pipeline report, we have written extensive documentation on all parts of running and interpreting the output of the pipeline. Given that a large fraction of aDNA researchers come from fields outside computational biology, such as social sciences - who often have limited computational training (such as J.A.F.Y), we have written documentation that also gives guidance on h ...
+Alongside the portable new pipeline report, we have written extensive documentation on all parts of running and interpreting the output of the pipeline. Given that a large fraction of aDNA researchers come from fields outside computational biology, and thus may have limited computational training, we have written documentation that also gives guidance on how to interpret each section of the repor ...
```
I was trying to emphasis many aDNA researchers is not from any super-technical discpline (chemists or engineers will often have some comp. training) who get 0% computational training except for word and excel. Can you suggest a rephrase?

TCLamnidis replied:
Isn't my existing suggestions covering that?
> "Given that a large fraction of aDNA researchers come from fields outside computational biology, **and thus may have limited computational training**, we have written documentation [...]"

jfy133 replied:
Fair enough. Maybe I'm going into too much 'PR' mode and selling it too hard...

TCLamnidis committed:
Update content/03.results-and-discussion.md

Co-authored-by: James A. Fellows Yates <jfy133@gmail.com>

TCLamnidis committed:
converted list sub-points to bullet points, markdown friendly

This way markdown renders these properly, and there aren't nested numbers in numbers.
